_posts/blog/2018-10-18-review-living-archive.md

```diff
@@ -0,0 +1,20 @@
+Review of the Living Archive of Aboriginal Languages
+
+Living Archive of Aboriginal Languages. Charles Darwin University et al, 2012, http://laal.cdu.edu.au/. Accessed 17 oct. 2018
+
+Simplicity is one of the key words for the Living Archive of Aboriginal Languages. This digital repository contains over 2500 books written in more than 25 indigenous languages of the Northern Territory of Australia, but the texts are available through a very easy-to-use interface displaying a clickable geographical map, with icons that redirect the user to the materials. Another key word for this collection is right there in its name: “Living”. The books are classified as “endangered literatures” on the website, thus the archive is set up to enhance their liveness by displaying them in an open access online platform. One of the main goals of this ongoing endeavor, as the “about” page states, is to connect these literatures to the communities they belong to. Why were these texts disconnected from their social origins in the first place? The answer is in the background story. 
```

Comment:
I might separate this into two  or even three paragraphs: one that begins "Another key word..." and a second that begins "Why were these texts..." That will help support the conversational tone of the writing.

Reply:
I don't think "but" is the correct way to connect the two ideas in the second sentence. Consider starting a new sentence instead.